routes/account.route.js

```diff
@@ -0,0 +1,87 @@
+import express from "express";
+import bcrypt from "bcryptjs";
+import accountService from "../services/account.service.js";
+import { v4 } from "uuid";
+
+const router = express.Router();
+
+router.get("/login", async function (req, res) {
+  res.render("vwAccount/login", {});
+});
+
+router.post("/login", async function (req, res) {
+  const user = await accountService.login(req.body.username);
+  if (user === null) {
+    return res.render("vwAccount/login", {
+      err_msg: "Invalid username or password.",
+    });
+  }
+  if (user.lockaccount === 1) {
+    return res.render("vwAccount/login", {
+      err_msg: "Your account has been banned",
+    });
+  }
+
+  const ret = bcrypt.compareSync(req.body.password, user.pass.toString());
+  if (ret === false) {
+    return res.render("vwAccount/login", {
+      err_msg: "Invalid username or password",
+    });
+  }
+
+  delete user.pass;
+
+  req.session.auth = true;
+  req.session.authUser = user;
+
+  let url = req.session.retUrl || "/";
+  delete req.session.retUrl;
+  if (user.permission === 1) {
+    // Redirect to Admin here
+  }
+  res.redirect(url);
+});
+
+router.post("/logout", function (req, res) {
+  req.session.auth = false;
+  req.session.authUser = null;
+
+  const url = req.headers.referer || "/";
+  res.redirect(url);
+});
+
+router.get("/register", async function (req, res) {
+  res.render("vwAccount/register", {});
+});
+
+router.post("/register", async function (req, res) {
+  const salt = bcrypt.genSaltSync(10);
```

Comment:
này nên tạo 1 biến toàn cục ở đầu file kiểu `const SALT_LENGTH = 10;` rồi pass cái biến vào đây chứ ko nên ghi số